slides/intro.md

```diff
@@ -0,0 +1,157 @@
+
+* More application-oriented and uses statistics
+
+* More computation
+
+* We have an bonus problem set on topic modeling in this course
+
+# What can we do with language?
+
+## Corpus linguistics
+
+* Statistics on a body of text <div class="column"><img src="img/lemmarank.png"></div>
+
+* Can use tool-generated information: morphology, syntax, semantics etc.
+
+* Arguably Kielipankki's main focus, we will do a project in this course
+
+# What can we do with language?
+
+## Modern large-scale machine learning
+
+* This is what you read about in the headlines <div class="column" style="width:30%"><img src="img/shoggoth.jpeg"></div>
+
+* Generic statistical approaches, no linguistic content
+
+* Huge amounts of data and computation, cutting-edge applications
+
+* A lot of this goes on at CSC, but we won't do it in this course
+
+# Kielipankki resources: Korp
+
+* A browser for corpora <div class="column" style="width:50%"><img src="img/korp_screenshot.png"></div>
+
+* Developed by Språkbanken (our Swedish sister organisation)
+
+* Advanced search features, supports some statistics views
+
+# Kielipankki resources: Corpora
+
+* Many modalities (text, speech, video, structured, scanned pages)
+
+* Some available on Korp, some as downloads, some as both
+
+* From completely open to very restricted
+
+* Text usually annotated with metadata, part of speech, base form, morphology, dependency syntax
+
+* We can also annotate named entities, sentiment, geospatial information etc.
+
+# Kielipankki resources: Tools
+
+* <a href="https://www.kielipankki.fi/tools/">https://www.kielipankki.fi/tools/</a>
+
+* Hosted tools: run on our machines, interact via the web (eg. WebAnno for manually annotating text)
+
+* Downloadable tools: tools maintained by us for that you can run on your own machine (eg. `finnish-tagtools`)
+
+* Software installed on our HPC platforms, need to log in there to use them
+
+# CSC resources: Data storage
+
+* HPC has a lot of capacity for temporary data
+
+* IDA, long-term storage for FAIR research data is hosted at CSC
+
+* Users have access to:
+    * an object storage system (Allas)
+	* a hosted database service (Kaivos)
+	* encrypted data storage via SD desktop
+
+# CSC resources: Cloud computing
+
+* Virtual servers (Pouta), you can host long-running services and websites here
+
+* Container cloud (Rahti), same but with containers
+
+* Notebooks, an instant programming environment
+
+# CSC resources: HPC (High-Performance Computing)
+
+* Access usually via a command line, or a browser-based app
+
+* **Puhti** -- General purpose. 682 CPU nodes, each with 40 cores + 80 Nvidia GPU nodes - we'll be using this through Jupyter in the browser!
+
+* **Mahti** -- Larger-scale jobs. 1404 CPU nodes, each with 128 cores, 180K CPU cores total! 24 GPU nodes, but very beefy ones (4 Nvidia A100's).
+
+* **Lumi** -- GPU-specialised jobs. Over 10K AMD GPUs. That's a lot of compute (#3 in the world!).
```

Comment:
Isn't Lumi going to have a number of partitions for different needs, only some of which have GPUs? Not that the GPU wouldn't be the impressive part, so if emphasis on it makes sense for the talk, it's okay to keep this as is.

Reply:
Yeah, true, but keeping it simple with this.